components/gh_events/GithubFeed.tsx

```diff
@@ -0,0 +1,102 @@
+"use client";
+import GitHubEvent from "@/components/gh_events/GitHubEvent";
+import { IGitHubEvent } from "@/lib/gh_events";
+import { useState } from "react";
+
+interface Filter {
+  title: string;
+  options: string[];
+}
+interface Props {
+  events: IGitHubEvent[];
+  filterEvetns: Filter[];
```

Comment:
fix this spelling mistake everywhere its repeating...

```suggestion
  filterEvents: Filter[];
```